dashboard/app/helpers/lti_helper.rb

```diff
@@ -0,0 +1,119 @@
+
+module LtiHelper
+
+  def windows_complex_password
```

Comment:
So for this method, we're generating a random password that has no 0s, 1s, vowels, or lowercase l, between 7 and 11 characters. And I think deleting the vowels is meant to remove the possibility of generating a word?

My understanding is that the user never actually sees this password, right? If thats the case, then I think some randomly generated UUID would be just as effective here.